From 549dc708794472fffce7974b33a1cab182034c61 Mon Sep 17 00:00:00 2001
From: Mike Hommey <glandium@debian.org>
Date: Tue, 7 Jul 2009 20:26:46 +0200
Subject: [PATCH 24/35] Don't export js_SetTraceableNativeFailed, which is only used internally

Otherwise, this is one more different symbol between JIT and non-JIT builds
---
 js/src/jsbuiltins.cpp |    2 +-
 js/src/jsbuiltins.h   |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/js/src/jsbuiltins.cpp b/js/src/jsbuiltins.cpp
index 29bb9f1..5232309 100644
--- a/js/src/jsbuiltins.cpp
+++ b/js/src/jsbuiltins.cpp
@@ -68,7 +68,7 @@ using namespace nanojit;
 
 extern jsdouble js_NaN;
 
-JS_FRIEND_API(void)
+void
 js_SetTraceableNativeFailed(JSContext *cx)
 {
     js_SetBuiltinError(cx);
diff --git a/js/src/jsbuiltins.h b/js/src/jsbuiltins.h
index 608853a..c1dd972 100644
--- a/js/src/jsbuiltins.h
+++ b/js/src/jsbuiltins.h
@@ -441,7 +441,7 @@ jsdouble FASTCALL
 js_BooleanOrUndefinedToNumber(JSContext* cx, int32 unboxed);
 
 /* Extern version of js_SetBuiltinError. */
-extern JS_FRIEND_API(void)
+extern void
 js_SetTraceableNativeFailed(JSContext *cx);
 
 extern jsdouble FASTCALL
-- 
1.7.0.rc0.69.g2ee80

